applications/CoSimulationApplication/custom_processes/data_transfer_3D_1D_process.h

```diff
@@ -0,0 +1,297 @@
+// KRATOS  / ___|___/ ___|(_)_ __ ___  _   _| | __ _| |_(_) ___  _ ___
+//        | |   / _ \___ \| | '_ ` _ \| | | | |/ _` | __| |/ _ \| '_  |
+//        | |__| (_) |__) | | | | | | | |_| | | (_| | |_| | (_) | | | |
+//         \____\___/____/|_|_| |_| |_|\__,_|_|\__,_|\__|_|\___/|_| |_|
+//
+//  License:         BSD License
+//                   license: CoSimulationApplication/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//
+
+#pragma once
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/define.h"
+#include "includes/model_part.h"
+#include "includes/kratos_parameters.h"
+#include "processes/process.h"
+
+/* The mappers includes */
+#include "spaces/ublas_space.h"
+#include "mappers/mapper_flags.h"
+#include "factories/mapper_factory.h"
+
+namespace Kratos
+{
+///@addtogroup CoSimulationApplication
+///@{
+
+///@}
+///@name Functions
+///@{
+
+/**
+ * @brief This method determines the 1D model part
+ * @param rFirstModelPart The first ModelPart
+ * @param rSecondModelPart The second ModelPart
+ * @return The 1D model part 
+ */
+ModelPart& Determine1DModelPart(ModelPart& rFirstModelPart, ModelPart& rSecondModelPart)
```

Comment:
can you move the logic to the source file?

Reply:
Okay